_posts/2017-10-08-rtimicropem.md

```diff
@@ -0,0 +1,71 @@
+---
+name: rtimicropem
+layout: post
+title: rtimicropem: Using an *R* package as platform for harmonized cleaning of data from RTI MicroPEM air quality sensors
+date: 2017-07-19
+authors:
+  - name: Maëlle Salmon
+    url: http://www.masalmon.eu/
+categories:
+  - blog
+tags:
+- R
+- community
+- software
+- review
+- hardware
+- ropenaq
+---
+
+As you might remember from [my blog post about `ropenaq`](https://ropensci.org/blog/blog/2017/02/21/ropenaq), I work as a data manager and statistician for an [epidemiology project called CHAI](http://www.chaiproject.org/) for Cardio-vascular health effects of air pollution in Telangana, India. One of our interests in CHAI is determining exposure, and sources of exposure, to PM2.5 which are very small particles in the air that have diverse adverse health effects. You can find more details about CHAI [in our recently published protocol paper](https://www.ncbi.nlm.nih.gov/pubmed/28606699). In this blog post that partly corresponds to the content of [my useR! 2017 lightning talk](http://sched.co/AxrS), I'll present a package we wrote for dealing with the output of a scientific device, which might remind you of similar issues in your experimental work.
+
+# Why write the `rtimicropem` package?
+
+![](/assets/blog-images/2017-07-19-rtimicropem/Allequipment_Frontview_cropped.jpg)
+
+Part of the CHAI project is a panel study involving about 40 people wearing several devices, as you see above. The devices includes a GPS, an accelerometer, a wearable camera, and a PM2.5 monitor outputting time-resolved data (the grey box on the left). Basically, with this device, the RTI MicroPEM, we get one PM2.5 exposure value every 10 second. This is quite exciting, right? Except that we have two main issues with it...
+
+![](/assets/blog-images/2017-07-19-rtimicropem/screenshot_output2.png)
+
+First of all, the output of the device, a file with a ".csv" extension corresponding to a session of measurements, in our case 24 hours of measurements, is not really a csv. The header contains information about settings of the device for that session, and then comes the actual table with measurements.
+
+Then, since the RTI MicroPEMs are nice devices but also a work-in-progress, we had some problems with the data, such as negative relative humidity. Because of these issues, we decided to write an R package whose three goals were to:
```

Comment:
maybe say "Second" instead of "Then", in order to follow from you saying there are two issues